### PR TITLE
chore: update user voyage roles in dev seed

### PR DESCRIPTION
**Pull request by Ajen07**
# Description

- Created `2 new users` and voyage `52` and `53`
- Added team `v52-tier1-team-1` to have only `PO` with   `memberId `as `41` and `42`
- Added team `v52-tier1-team-2` to have only `SM`  with `member Id`  `43` and `44`
- Added team `v52-tier2-team-2` to have  `3 developers` with `memberId` `45`, `46` and `47`
- Added team `v52-tier2-team-4`  to have    `1 UI/UX designer`  , `1 data scientist` and `1 voyage guide`  having `memberId `as `48`, `49` and `50`
- Added team `v52-tier3-team-3`  to have  all roles `1 SM`, `1 PO`,` 3 developers`, `1 UI/UX designer` , `1 data scientist`  and `1 voyage guide` having `memberId `as  `51`  , `52`, `53`, `54`, `55`, `56`, `57` and `58`

## Issue link

[Fixes # (issue)](https://app.clickup.com/t/86b27xvk2)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Feature updates / changes
- [ ] Tests
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# How Has This Been Tested?

- tested with `yarn seed`
- tested with `yarn test:docker`

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have updated the change log

### CODE REVIEW

**Review by cherylli — COMMENTED** [review]
I think it's easier if all the teams with different composition is in the same voyage, so we don't have to manually overwrite the dates to check, but I need to check if frontend can handle that first.

**Comment by Ajen07** [comment]
> I think it's easier if all the teams with different composition is in the same voyage, so we don't have to manually overwrite the dates to check, but I need to check if frontend can handle that first.

In that case we will have people  belonging to different teams in the same voyage

**Comment by cherylli** [comment]
> > I think it's easier if all the teams with different composition is in the same voyage, so we don't have to manually overwrite the dates to check, but I need to check if frontend can handle that first.
> 
> In that case we will have same people belonging to different teams in the same voyage or we have to create more users.

yes I prefer just using the same members, so we don't have to create too many, but lets see what frontend says. Eventually they will have to handle this situation, but I don't know if the current frontend can handle it yet

**Review by JoshuaHinman — DISMISSED** [review]
all tests passed

**Comment by cherylli** [comment]
@Ajen07, I'd say just go ahead and make them all in the same voyage, v52, 53 sounds good since they are coming up. I just saw the current seed has people in more than 1 team anyway

**Review by cherylli — CHANGES_REQUESTED** [review]

File: prisma/seed/voyage-teams.ts [hunk 1]
```diff
@@ -1050,6 +1050,487 @@ export const populateVoyageTeams = async () => {
         },
     });
 
+    //v52
+
+    // team with only PO
```
[the remark this answers is not in the document]
Comment (Ajen07) [reply prisma/seed/voyage-teams.ts:1055]: Ok I will do the necessary changes

**Review by JoshuaHinman — DISMISSED** [review]
All tests passed!

File: prisma/seed/voyage.ts [hunk 2]
```diff
@@ -88,4 +88,34 @@ export const populateVoyages = async () => {
...
+    await prisma.voyage.create({
+        data: {
+            number: "53",
+            status: {
+                connect: {
+                    name: "Upcoming",
+                },
+            },
+            startDate: new Date("2025-01-06T17:00:00.000Z"),
+            endDate: new Date("2025-02-16T04:59:59.000Z"),
```
Comment (cherylli) [inline prisma/seed/voyage.ts:115]: please update endDate, soloprojectDeadline, CertificationIssueDate and showcasePublishDat as above (v52) - you can see here for more detail (https://github.com/chingu-x/chingu-dashboard-be/wiki/Timezone)

**Review by cherylli — CHANGES_REQUESTED** [review]
Please also update the wiki page with the new teams
https://github.com/chingu-x/chingu-dashboard-be/wiki/Test-Teams-and-Users

**Comment by cherylli** [comment]
update second reviewer to @curtwl as josh is away

**Review by cherylli — CHANGES_REQUESTED** [review]
> Please also update the wiki page with the new teams https://github.com/chingu-x/chingu-dashboard-be/wiki/Test-Teams-and-Users

Still need this updated

I fixed the voyage dates based on the voyage schedule, it is very important we have the date and exact time right as it would impact what pages/features is enabled/disabled in the frontend

**Comment by Ajen07** [comment]
> > Please also update the wiki page with the new teams https://github.com/chingu-x/chingu-dashboard-be/wiki/Test-Teams-and-Users
> 
> Still need this updated
> 
> I fixed the voyage dates based on the voyage schedule, it is very important we have the date and exact time right as it would impact what pages/features is enabled/disabled in the frontend

I have updated the wiki page

**Review by cherylli — APPROVED** [review]

**Comment by cherylli** [comment]
Updated v53 start date as it was an invalid date.
Seeding worked fine

**Review by curtwl — APPROVED** [review]
lgtm

**Pull request merged**